### DISCUSSION TITLE
redwoodjs

### DISCUSSION THREAD

**Discussion by bickmark** [ben-rogerson/twin.examples#30] [Q&A] [answered]
Has anyone had success integrating twin.macro into redwoodjs? Having some issues. Thanks.

**Comment by ben-rogerson** [comment, marked as answer]
I'm interested in this also - have you got a half set up repo we could work from?

**Reply by bickmark** [reply]
Someone in the RedwoodJS Discourse Forums posted a solution. I just tried it out and it worked for me. Here is the link: https://community.redwoodjs.com/t/twin-macro-integration/1746/3?u=bickmark

**Reply by ben-rogerson** [reply]
great! thanks for the link
